source/recipe-individual/scripts/individualCustom.js

```diff
@@ -0,0 +1,295 @@
+            margin-right: 0vw;
+            margin-left: 83%;
+            padding-left: 3vw;
+            top: .2em;
+            background-color:none;
+          }
+          .user-portal{
+            position: absolute;
+            justify-self: right;
+            margin-right: 0vw;
+            margin-left: 88%;
+            padding-left: 3em;
+            top: 1em;
+            background-color:none;
+          }
+          .logoImg {
+           position:fixed;
+           justify-items: end;
+           padding-left: .2em;
+           padding-right: 5em;
+           top: 1.2em;
+           width: 8%;
+           left: 0vw;
+          }
+          .logoText {
+            position:absolute;
+            height: 10em;
+            width: 15em;
+            justify-items: begin;
+            padding-left: 2.6em;
+            padding-right: 8em;
+            top: 0em;
+            margin-top: 0%;
+            padding-top: .2%;
+            left: 0vw ;
+            font-style: oblique;
+          }
+          h4{
+            font-family: 'Abril Fatface', cursive;
+            font-size: 5vh;
+            text-decoration: wavy;
+            color: black;
+            margin-left: 8%;
+            top: 0em;
+            margin-top: 9%;
+            padding-top: 0%;
+          }
+          h3{
+            font-family: 'Abril Fatface', cursive;
+            width: 8vw;
+            justify-self: left;
+          }
+          p {
+            font-family: 'Abril Fatface', cursive;
+          }
+
+          li {
+              font-family: 'Abril Fatface', cursive;
+          }
+
+          h2 {
+            font-family: 'Abril Fatface', cursive;
+          }
+        `;
+
+        styleElem.innerHTML = styles;
+        
+        const container = document.createElement("div");
+
+        const rightContainer = document.createElement('div');
+        rightContainer.setAttribute('class', 'application-type');
+        const thumbnail = document.createElement("img");
+        thumbnail.setAttribute("src",data.image);
+        const ingredientsContainer = document.createElement('div');
+        ingredientsContainer.setAttribute("class", 'essay-text');
+        const ingredientsHeadline = document.createElement('h2');
+        ingredientsHeadline.innerHTML = "Ingredients";
+        const ingredientsText = document.createElement('ul');
+        for(var i = 0; i < data.extendedIngredients.length; i++) {
+            const single = document.createElement("li");
+            single.innerHTML = data.extendedIngredients[i].original;
+            ingredientsText.appendChild(single);
+        }
+        rightContainer.appendChild(thumbnail);
+        ingredientsContainer.appendChild(ingredientsHeadline);
+        ingredientsContainer.appendChild(ingredientsText);
+        rightContainer.appendChild(ingredientsContainer);
+
+        const topMiddleContainer = document.createElement('div');
+        topMiddleContainer.setAttribute("class", 'image-box');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445170)
```suggestion
        topMiddleContainer.setAttribute("class", "image-box");
```